### PR TITLE
curlcpp: 20160901 -> 1.0

### PR DESCRIPTION
**Pull request by rszibele**
###### Motivation for this change

Version bump and updated my maintainer info.

For changes in curlcpp see https://github.com/JosephP91/curlcpp/compare/98286da1d6c9f6158344a8e272eae5030cbf6c0e...f3c5499b312ba649b87f205e8c1f771958227a05 .

###### Things done

- [X] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by rvl** [comment]
Seems find considering there are no nixpkgs which depend on this lib.

Would be nice if this project had releases. Just now there is a new commit on [JosephP91/curlcpp#bf0b97d48c4b17e5dc96ddfa0eabd4c2c9a74b21](https://github.com/JosephP91/curlcpp/commits/master).

**Comment by rszibele** [comment]
@rlv I've opened an issue over at curlcpp regarding releases.

File: pkgs/development/libraries/curlcpp/default.nix [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+stdenv.mkDerivation rec {
+  name = "curlcpp-${version}";
+  version = "1.0";
 
   src = fetchFromGitHub {
     owner = "JosephP91";
     repo = "curlcpp";
-    rev = "f3c5499b312ba649b87f205e8c1f771958227a05";
-    sha256 = "0mwj5ghr04s6n3dkm50gp1aj0k30k3g4jkhhvq28vbqa0rmkyxk7";
+    rev = "bf0b97d48c4b17e5dc96ddfa0eabd4c2c9a74b21";
```
Comment (joachifm) [inline pkgs/development/libraries/curlcpp/default.nix]: You can refer to tags as well, which is preferable if there is one associated with the release.

Comment (rszibele) [inline pkgs/development/libraries/curlcpp/default.nix]: Cool :+1:. Updated the commit.

**Review by rvl — APPROVED** [review]

**Comment by joachifm** [comment]
Thank you

**Pull request merged**